README.md

```diff
@@ -602,10 +602,12 @@ we can use stack defined as follows:
 stacks:
   us-east-1:
     my-stack:
-      sparkle_pack_template: template_name
+      template: template_name
+      compiler: sparkle_formation
```

Comment:
Did you consider adding the compiler type to the `compiler_options` properties? If so, I'm curious to understand why you chose to not to.

Reply:
Yeah I briefly considered that. I went with a specific attribute on StackDefinition because it's a value for StackMaster, not the compiler itself.